setup.py

```diff
@@ -2,31 +2,34 @@
 import os
 from setuptools import setup, find_packages
 
-import censys
-
 NAME = "censys"
-DESCRIPTION = censys.__doc__.strip()
+DESCRIPTION = "An easy-to-use and lightweight API wrapper for the Censys Search Engine (censys.io)."
 GIT_URL = "https://github.com/censys/censys-python"
 ISSUE_URL = GIT_URL + "/issues"
 DOC_URL = "https://censys-python.rtfd.io"
 
+here = os.path.abspath(os.path.dirname(__file__))
+
+pkg_vars = {}
+
+with open(os.path.join(here, NAME, "version.py")) as fp:
+    exec(fp.read(), pkg_vars)
```

Comment:
Can we not just import that file? This feels... weird

Reply:
We actually cannot, it is not importable before installing it

Reply:
It seems like it was imported before though? `import censys; censys.__version__` Was that some black magic?

If this is the best/only way, though, this is fine, it just feels icky

Reply:
Yeah its a little icky, but the way it was working before was building based on the local version.